notebooks/marimo/Exhibit_2_Variance_Plot.py

```
@@ -0,0 +1,242 @@
+@app.cell
+def _():
+    import ray
+
+    ray.init()
+    return (ray,)
+
+
+@app.cell
+def _(np):
+    def legend_thick(ax, *args, **kwargs):
+        leg = _ax.legend(*args, **kwargs)
```

Ruff (0.15.10)

[error] Undefined name `_ax`

(F821)